doc/user_guide/getting_started.rst

```diff
@@ -221,9 +221,58 @@ in the Signal2D subclass.
     >>> im
     <Signal2D, title: , dimensions: (30|20, 10)>
 
-Note the HyperSpy rearranges the axes position to match the following pattern:
-(navigatons axis 0,..., navigation axis n|signal axis 0,..., signal axis n).
-This is the order used for :ref:`indexing the BaseSignal class <signal.indexing>`.
+Note that HyperSpy rearranges the axes when compared to the array order. The
```

Comment:
Yes. The only case when it ends up the same is with one navigation and signal axis each, though